### DISCUSSION TITLE
Importing and exporting in ES6 using module.exports

### DISCUSSION THREAD

**Discussion by Rahma-AlWadhahi** [EddieHubCommunity/support#1138] [Tips and Tricks]
I found a question in stackoverflow which addresses a similar question that has been raised by @janaSunrise in discord:

“ Why do I get this error even if I actually export it...”
![4AD1A573-E21D-4DED-9AC3-9C5F8F344019](https://user-images.githubusercontent.com/61970868/111956710-81475d80-8b04-11eb-863e-b26dc1959881.png)
![F85C78FB-47CD-422B-B5BB-766E730840E4](https://user-images.githubusercontent.com/61970868/111956715-83a9b780-8b04-11eb-9796-e464f850edcc.png)

### Solution 
[Module.exports and ES6 Import](https://stackoverflow.com/questions/34278474/module-exports-and-es6-import)

**Comment by janaSunrise** [comment]
Ohhh, great!! Thank you so much!

**Reply by Rahma-AlWadhahi** [reply]
You’re welcome. I found another one that talks about multiple things to export

[Declare multiple module exports in nodejs](https://stackoverflow.com/questions/16631064/declare-multiple-module-exports-in-node-js)
